sequencer/txselector/txselector.go

```diff
@@ -112,3 +112,14 @@ func NewTxSelector(
 		// coordAccount:    coordAccount,
 	}, nil
 }
+
+// LocalAccountsDB returns the LocalStateDB of the TxSelector
+func (txsel *TxSelector) LocalAccountsDB() *statedb.LocalStateDB {
+	return txsel.localAccountsDB
+}
```

Comment:
@Intizar-T I'm not sure that `txSelector` has local connection with only accountsDB. Could you elaborate what this part is for and what about Vouch or Score?